### PR TITLE
End of Sprint-8 merging everything from develop to main.

### PR DESCRIPTION
**Pull request by abirshah**
We are merging everything from the pre-main to the main repo.

### CODE REVIEW

File: Pet_Detection/pet_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    plt.imshow(my_img)
+    height, width, _ = my_img.shape
+    blob = cv2.dnn.blobFromImage(my_img, 1 / 255, (416, 416), (0, 0, 0), swapRB=True, crop=False)
+    net.setInput(blob)
+    boxes, confidences, class_ids = pd.getNumbers(net, width, height)
+    indexes = cv2.dnn.NMSBoxes(boxes, confidences, .5, .4)
+    if len(indexes) > 0:
+        for i in indexes.flatten():
+            pd.draw_bounding_boxes(boxes=boxes, index=i, classes=classes, class_ids=class_ids,
```
Comment (MikeG3) [inline Pet_Detection/pet_test.py:24]: Sonar cloud pointing to this line of code, it seems to be missing a parameter

**Review by MikeG3 — COMMENTED** [review]
### _Overall Remarks._
I was able to run all this code properly, without issues

Everything that required refactoring, and editing is completed.

All code for this Sprint has been merged as needed.

Only issue, is a bug reported by sonar cloud, in Pet_Detedction/pet_test.py
I havent ran this particular section of code, perhaps it can be corrected before or after the final merge
Please see the inline comment, at line 24

File: Pet_Detection/pet_test.py (hunk 1 above)
Comment (deeppatel-11) [reply Pet_Detection/pet_test.py:24]: I don't know why its saying that but, its actually not missing any

**Comment by deeppatel-11** [comment]
Approving the Merge of develop to the main branch at the end of sprint 8

**Review by deeppatel-11 — APPROVED** [review]

**Pull request merged**